### DISCUSSION TITLE
Welcome to KD_Lib Discussions!

### DISCUSSION THREAD

**Discussion by Het-Shah** [SforAiDl/KD_Lib#101] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mnsbharadwaj** [comment]
Can you please provide one end to end example with teacher and student model using this lib.

**Reply by Het-Shah** [reply]
You can check out the [docs](https://kd-lib.readthedocs.io/en/latest/index.html), we have added some tutorials in the [tutorials section](https://kd-lib.readthedocs.io/en/latest/usage/tutorials/index.html). For example, Vanilla KD algorithm's tutorial [link](https://kd-lib.readthedocs.io/en/latest/usage/tutorials/VanillaKD.html)
